tensorflow/core/kernels/depthwise_conv_op_gpu.h

```diff
@@ -1758,14 +1758,13 @@ void LaunchDepthwiseConvBackpropFilterOp<GpuDevice, T>::operator()(
     const T* input, T* filter_backprop, TensorFormat data_format) {
   auto stream = ctx->op_device_context()->stream();
 
-  // By disabling this exception, we can discover if other determinism
-  // exceptions are reached in the execution of a given model.
+  // It's simpler to catch this here than in
+  // DepthwiseConv2dNativeBackpropFilterOp
   OP_REQUIRES(
-      ctx,
-      !OpDeterminismRequired() || DisableDepthwiseConvDeterminismExceptions(),
-      errors::Unimplemented(
+      ctx, !OpDeterminismRequired(), errors::Unimplemented(
           "A deterministic GPU implementation of DepthwiseConvBackpropFilter is"
-          " not currently available."));
+          " not available with this version of cuDNN. Use version 7.6.3 or"
```

Comment:
The error should state "Please build with cuDNN version 7.6.3 or newer". Since `CUDNN_VERSION` checks the version TF is built with, even if the cuDNN version at runtime is newer.

Luckily our pip packages are built with cuDNN 8.1 so this never will occur for the prebuilt pip packages.